collect_app/build.gradle

```diff
@@ -189,6 +189,10 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
 
+    kotlinOptions {
```

Comment:
I added this to use doOnTextChanged in https://github.com/getodk/collect/pull/4591/files#diff-651f8f96000eb2c3ca961c1f82e7a5300b5cc835f97d7ee77762fe83048eeebeR48

which is an extension function and a nice replacement for `addTextChangedListener` (`beforeTextChanged`, `onTextChanged`, `afterTextChanged`) which requires overriding three methods what doesn't make sense if you need just one.

Reply:
Ah sorry I didn't see what you mean at first but you needed it because you can't use an `inline` extension for Java 8 code while targeting 6 with Kotlin right?

Reply:
Yes